lib/clusterManager.js

```diff
@@ -45,6 +45,7 @@ const workerChangeDecisionDelay = process.env.workerChangeDecisionDelay || 4;
 
 var clusterLogger = zluxUtil.loggers.clusterLogger;
 
+// Inserting the Zowe message code inline due to message resource file not loaded yet
 clusterLogger.info("ZWED0211I - The number of processors is: " + cpuCount);
 if (minWorkers != process.env.ZLUX_MIN_WORKERS) {
   clusterLogger.info("ZWED0212I - Environmental variable ZLUX_MIN_WORKERS was not a valid number therefore " + minWorkers + " will be used as the minimum workers");
```

Comment:
Do we not have a variable reference to "ZLUX_MIN_WORKERS" string?

Reply:
Not relevant to logging changes